tests/commands/DeployCommand.test.js

```diff
@@ -119,7 +119,7 @@ describe('DeployCommand', () => {
         ]
 
         for (let { label, input, output } of cases) {
-          expect(command.indexAdapter(input), label).toEqual(output)
+          expect(command.indexAdapter(input)).toEqual(output)
```

Comment:
looks like the label bits are now dead code. Let's delete them. That's true for lines such as 101 as well.

Reply:
Replaced all the label cases things in ccb8573

That commit puts all those tests in individual expect calls, so errors will actually produce a useful stacktrace.